dsgrid/config/representative_period_time_dimension_config.py

```diff
@@ -60,6 +130,7 @@ def get_timestamp_load_data_columns(self):
         return self._format_handler.get_timestamp_load_data_columns()
 
     def get_tzinfo(self):
+        # TBD
```

Comment:
Should this just return TimeZone.LOCAL?

Reply:
No, the output of this func needs to be a real tzinfo value

Reply:
So will it end up always returning None? Or would you want it to return a mapping of ids of some sort to prevailing time zone objects?